### DISCUSSION TITLE
在initialValue 的时候不触发 onFieldValueChange，应该怎么做？

### DISCUSSION THREAD

**Discussion by Zhou-Bill** [alibaba/formily#1089] [Ideas]
我希望在initialValue 的时候不触发 onFieldValueChange，但是希望某个字段触发会引发联动，应该怎么做？
后续会用到onFieldValueChange 做联动。。

[demo](https://codesandbox.io/s/fast-leaf-cm3dh?file=/index.js)

**Comment by janryWang** [comment]
触发是肯定会触发的。建议改成onFieldInputChange
